### ISSUE TITLE
False 406 response code on InvalidArgumentException

### ISSUE THREAD

**Issue by alexander-schranz** [sulu/sulu#4568]
| Q | A
| --- | ---
| Bug? | no
| New Feature? | no
| Sulu Version | 1.6.26, 2.0.0-alpha6
| Browser Version | -

#### Actual Behavior

If there is any InvalidArgumentException when calling the WebsiteController this is catched and replaced by [Client Error 406](https://github.com/sulu/sulu/blob/1.6.26/src/Sulu/Bundle/WebsiteBundle/Controller/WebsiteController.php#L73) not acceptable.

<img width="1038" alt="Bildschirmfoto 2019-05-15 um 18 51 39" src="https://user-images.githubusercontent.com/1698337/57793810-82f2ae00-7742-11e9-8318-39530981c692.png">

#### Expected Behavior

I think we should not replace all InvalidArgumentExceptions with a 406 Client Error. If an exception really happens not try to catch it and let it go so its logged and under develop a correct stacktrace is shown. 

<img width="1034" alt="Bildschirmfoto 2019-05-15 um 18 51 06" src="https://user-images.githubusercontent.com/1698337/57793814-871ecb80-7742-11e9-8204-79a777b17681.png">

#### Steps to Reproduce

E.g. use a not exist content type.

#### Possible Solutions

Remove the try and catch.

**Comment by alexander-schranz** [comment]
@sulu/core-team any case where this is really a Client Error (Preview?) and when yes can we throw there a more specific exception. /cc @wachterjohannes

**Comment by danrot** [comment]
If there is a better exception to be caught if a template is not found for the given file type, then I would definitely go for it. If twig itself only throws an `InvalidArgumentException` then we would have to parse the error message somehow... But that would definitely need a good test, not that this is not working again if they just change the error message.

**Comment by alexander-schranz** [comment]
@danrot ok. What about doing a check before?

```php
        if (!$this->get('twig')->getLoader()->exists($viewTemplate)) {
            throw new HttpException(406, 'Template does not exist in this format.');
        }
```

**Comment by danrot** [comment]
Would also work for me.

**Comment by alexander-schranz** [comment]
This was implemented in #4572
